packages/core/src/bin/commands/dev.ts

```diff
@@ -1,6 +1,7 @@
 import { existsSync } from "node:fs";
 import path from "node:path";
-import { type BuildResult, BuildService } from "@/build/service.js";
+import { createBuildService } from "@/build/index.js";
+import { type BuildResult } from "@/build/service.js";
```

Comment:
Might make sense to import this from `build/index.js`

Reply:
yes